NARFO_BE/NARFO_BE/encryption.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NARFO_BE
+{
+    public class encryption
+    {/*
+        public static String sha256_hash(string value)
+        {
+            StringBuilder Password = new StringBuilder();
+
+            using (var hash = SHA256.Create())
+            {
+                Encoding enc = Encoding.UTF8;
+                Byte[] result = hash.ComputeHash(enc.GetBytes(value));
+
+                foreach (Byte b in result)
+                    Password.Append(b.ToString("x2"));
+            }
+
+            return Password.ToString();
+        }*/
+        
+
+
+       public static byte[] SaltBytes { get; private set; }
+        public enum Supported_HA
+        {
+            SHA256
```

Comment:
Look at PBKDF2, SHA256 is way too fast.
https://www.owasp.org/index.php/Using_Rfc2898DeriveBytes_for_PBKDF2

Reply:
already on it ... we'll be using **PBKDF2** from now..